tests/components/create_team/display_name.test.jsx

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) 2018-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+import React from 'react';
+import {shallow} from 'enzyme';
+import {FormattedMessage} from 'react-intl';
+
+import {mountWithIntl} from 'tests/helpers/intl-test-helper';
+import DisplayName from 'components/create_team/components/display_name.jsx';
+import Constants from 'utils/constants.jsx';
+
+jest.mock('images/logo.png', () => 'logo.png');
+
+describe('/components/create_team/components/display_name', () => {
+    global.window.mm_license = {};
+    global.window.mm_config = {};
```

Comment:
Could you please remove this global config (same with other tests)? Global config was recently updated.